media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# requires proaudio overlay
+
+EAPI=5
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+SRC_URI=""
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE=""
+
+DEPEND="virtual/jack
+	media-libs/freetype
+	<dev-util/premake-4.3"
```

Comment:
Also the project website mentions more deps:

> ALSA
> freetype2
> OpenGL/Mesa
> X11 core and extensions (XShm, XRender and XCursor)

Are these really not required? Or are they optional? If they're optional they should be USE flags to avoid automagic dependencies.